purchases/src/main/kotlin/com/revenuecat/purchases/CustomerInfoUpdateHandler.kt

```diff
@@ -36,12 +39,21 @@ internal class CustomerInfoUpdateHandler(
 
     private var lastSentCustomerInfo: CustomerInfo? = null
 
+    private val _customerInfoStateFlow = MutableStateFlow<CustomerInfo?>(null)
+
+    init {
+        _customerInfoStateFlow.value = getCachedCustomerInfo(identityManager.currentAppUserID)
+    }
```

Comment:
I think this should work?
```suggestion
    private val _customerInfoStateFlow = MutableStateFlow<CustomerInfo?>(
        getCachedCustomerInfo(identityManager.currentAppUserID)
    )
```